### DISCUSSION TITLE
Turn workflows on/off as part of a deployment

### DISCUSSION THREAD

**Discussion by StylezDevops** [microsoft/powerplatform-build-tools#454] [Ideas]
@kkanakas

FEATURE REQUEST:

Currently, I turn workflows on or off as needed when deploying a solution with a post deployment PowerShell task. An official Build Tools task to accomplish this would be fantastic. Perhaps an addition to the deployment-settings.json file could be implemented where one provides a solution name and the list of workflow names within this solution - along with the required state (on/off).

Kind Regards
Dan Styles

**Comment by jboliveira** [comment]
Furthermore, it would be ideal if we could limit the turn off/on procedure to only the new and updated flows, rather than all.
It would speed up the operations much more in cases with 200 or more flows.
At the moment, I'm carrying out these tasks using a custom PowerShell script job that is performed immediately after deployment.

**Reply by StylezDevops** [reply]
@jboliveira I also do the same. Excellent addition to the discussion. Thanks

**Comment by rajyraman** [comment]
@StylezDevops - This seems to be hidden behind a flag in featureflags.json at the moment. Haven't tried it.

![image](https://github.com/microsoft/powerplatform-build-tools/assets/5035266/0ae7a7e5-c102-4595-954a-835c286cf0c7)

![image](https://github.com/microsoft/powerplatform-build-tools/assets/5035266/c5389b7f-eef3-4853-b408-9e916bf61b85)

**Reply by StylezDevops** [reply]
I saw a workflows addition to the solution settings file exported by PAC cli a few versions back but it disappeared with the next version. Seems like this is a feature that had been thought about and planned
